styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,87 @@
 .footer{
-  
-}
\ No newline at end of file
+  ul{
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+  .footer-menu {
+    li {
+      list-style: none;
+      padding: 0;
+      margin: 8px 0;
+      a {
```

Comment:
We probably should either move this a outside of the li (directly under .footer-menu), or perhaps even assign classes to the a links and take it out of .footer-menu altogether. Let's get @bighappyface's feedback on that.